v3/src/components/case-table/case-table.scss

```
@@ -13,6 +13,7 @@ $table-body-font-size: 8pt;
     height: 100%;
     display: flex;
     overflow-x: auto;
+    background-color: #FFFFFF;
```

```suggestion
    background-color: white;
```
A bit more readable than hex colors.